### ISSUE TITLE
Misinformation in the FS.init() docs

### ISSUE THREAD

**Issue by sgbeal** [emscripten-core/emscripten#17117]
The FS.init() docs at:

<https://emscripten.org/docs/api_reference/Filesystem-API.html#FS.init>

say that the input proxy function "should return an ASCII character code..." and that the output proxies "will be called with an ASCII character code whenever the program writes..."

That cannot possibly be correct: ASCII is limited to the range 0-127, inclusive, and emscripten-compiled binaries have no problems outputting non-ASCII UTF-8 characters (e.g. emoji), all of which have bit 8 set (and are thus greater than 127).

_Presumably_ the intent of those docs is that the functions must return an integer in the range 0..255, inclusive?

**Comment by sbc100** [comment]
Yes, you are correct, those callback can return arbitrary bytes (integers in the range 0..255).

However, I do wonder if anyone is actually using those paramaters.  All of our examples set `Module['stderr']` etc instead.  I wonder if we can remove that codepath.
